tests/integration/targets/ntnx_ndb_time_machines_with_multi_clusters/tasks/data_access_management_and_snapshots.yml

```diff
@@ -0,0 +1,245 @@
+---
+- debug:
+    msg: Start testing ntnx_ndb_time_machine_clusters
+
+- name: create data access instance with cluster name and sla name
+  ntnx_ndb_time_machine_clusters:
+    tm_uuid: "{{time_machine.uuid}}"
+    cluster:
+      name: "{{cluster.cluster2.name}}"
+    sla:
+      name: "{{sla.name}}"
+  register: out
+  ignore_errors: true
+
+
+
+- name: check listing status
+  assert:
+    that:
+      - out.response is defined
+      - out.tm_uuid is defined
+      - out.changed == true
+      - out.cluster_uuid is defined
+      - out.failed == false
+    fail_msg: "fail: Unable create data access instance with cluster name and sla name"
+    success_msg: "pass: create data access instance with cluster name and sla name finished successfully"
+#######################
+- name: update data access instance with new sla name
+  ntnx_ndb_time_machine_clusters:
+      tm_uuid: "{{time_machine.uuid}}"
+      cluster:
+        name: "{{cluster.cluster2.name}}"
+      sla:
+        name: "{{sla2.name}}"
+  register: result
+  ignore_errors: true
+
+- debug:
+    var: result
+
+- name: check listing status
+  assert:
+    that:
+      - result.response is defined
+      - result.tm_uuid is defined
+      - result.changed == true
+      - result.cluster_uuid is defined
```

Comment:
Please check the cluster_uuid correctness as well here. For example, - result.cluster_uuid == cluster.cluster2.uuid